src/msw/overlay.cpp

```diff
@@ -162,7 +162,18 @@ void wxOverlayImpl::Init(wxDC* dc, int , int , int , int )
 
     // The rectangle must be in screen coordinates
     m_rect = m_window->GetClientSize();
-    m_window->ClientToScreen(&m_rect.x, &m_rect.y);
+
+    if ( m_window->GetLayoutDirection() == wxLayout_LeftToRight )
+    {
+        m_window->ClientToScreen(&m_rect.x, &m_rect.y);
```

Comment:
Should we perhaps always use the version in the `else` branch?

Or, alternatively, change the basic `ClientToScreen()` itself?